### DISCUSSION TITLE
检测、分类、识别的推理模型已分别下载解压，但尝试运行推理脚本失败，无返回结果

### DISCUSSION THREAD

**Discussion by X17exe** [PaddlePaddle/PaddleOCR#12183] [Q&A]
请提供下述完整信息以便快速定位问题/Please provide the following information to quickly locate the problem

- 系统环境/System Environment：Windows 10
- 版本号/Version：
    - Paddle：2.5.1  
    - PaddleOCR：2.7.0.2 
- 问题相关组件/Related components：predict_system.py
- 运行指令/Command Code：
  ```bash
  python3 tools/infer/predict_system.py --image_dir="./doc/imgs/example.jpg" --det_model_dir="./ch_PP-OCRv4_det_infer/" --cls_model_dir="./ch_ppocr_mobile_v2.0_cls_infer/" --rec_model_dir="./ch_PP-OCRv4_rec_infer/" --use_angle_cls=true
  ```
- 完整报错/Complete Error Message：
   无报错，运行完指令就直接结束进入等待新指令输入状态了，文档中说结果会保存在./inference_result/中，也没有找到这个目录。

**Comment by X17exe** [comment]
example.jpg是我自己加入目录中的测试图。

**Comment by SWHL** [comment]
你好，请用现有仓库最新代码再次尝试。
